libs/StatusQ/tests/CMakeLists.txt

```diff
@@ -0,0 +1,44 @@
+cmake_minimum_required(VERSION 3.5)
+
+project(TestStatusQ LANGUAGES CXX)
+
+enable_testing()
+
+set(QT_NO_CREATE_VERSIONLESS_FUNCTIONS true)
+
+find_package(Qt6 6.3 COMPONENTS Quick Qml QuickTest REQUIRED)
```

Comment:
CMake should inherit varibles from root CMakeLists.txt. We can specify Qt version, cxx standard and may others just in root file OR in specific configuration file which will be much easy to maintain.

Reply:
I agree with you; this is the right way. I will improve it later on.